src/components/liveblog/body.tsx

```diff
@@ -22,32 +25,43 @@ const LiveBodyStyles = (pillarStyles: PillarStyles): SerializedStyles => css`
 interface LiveblogBodyProps {
     pillar: Pillar;
     blocks: LiveBlock[];
-    imageSalt: string;
+    totalBodyBlocks: number;
+    imageMappings: ImageMappings;
 }
 
-const LiveblogBody = ({ pillar, blocks, imageSalt }: LiveblogBodyProps): JSX.Element => {
+const LiveblogBody = (props: LiveblogBodyProps): JSX.Element => {
+    const { pillar, blocks: initialBlocks, imageMappings, totalBodyBlocks } = props;
+    const [blocks] = useState(initialBlocks);
 
-    const initialBlocks = blocks.slice(0, 7);
-    const LoadMore = ({ total }: { total: number }): JSX.Element | null => total > 10
-        ? <LiveblogLoadMore pillar={pillar}/> 
+    async function loadMoreBlocks(): Promise<void> {
+        const response = await fetch('?date=2020-03-09T11%3A11%3A49Z&filter=newer');
```

Comment:
Hardcoded date?

Reply:
I'm going to limit this PR to the rehydration and then after look at the mapi endpoint and fetching